src/placeos-driver/proxy/remote_driver.cr

```diff
@@ -231,12 +212,8 @@ module PlaceOS::Driver::Proxy
     def debug
       module_id = module_id?
       raise Error.new(ErrorCode::ModuleNotFound, "could not find module id", *@error_details) unless module_id
-
-      core_uri = which_core(module_id)
-
-      # build request
-      core_uri.path = "/api/core/v1/command/#{module_id}/debugger"
-      HTTP::WebSocket.new(core_uri)
+      client = Core::Client.new(which_core)
+      client.debug(module_id)
```

Comment:
```debug(module_id : String, &block : String ->)```

This method accepts a block. It will also block. Block block block.
We want it to block though, general tip for async code is to expose synchronous interfaces.